app/code/Magento/AsynchronousOperations/Model/BulkManagement.php

```diff
@@ -168,7 +168,7 @@ public function retryBulk($bulkUuid, array $errorCodes)
                     $currentBatchSize = 0;
                 }
                 $currentBatchSize++;
-                $operationIds[] = $operation->getId();
+                $operationIds[] = $operation->getOperationKey();
```

Comment:
That's not oberation ids anymore. Pls rename it in all places
```suggestion
                $operationKeys[] = $operation->getOperationKey();
```